content/topics/solo-learn/sql-intermediate/1-data-manipulations/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+  - topics/solo-learn/introduction-to-sql/4-data-analysis/_index.md

```

Comment:
this does not need to point to a file(`_index.md`) just like the rest of the content